src/endpoints/cart.js

```diff
@@ -36,8 +36,11 @@ class CartEndpoint extends BaseExtend {
     return this.call
   }
 
-  AddProduct(productId, quantity = 1, data = {}) {
-    const body = buildCartItemData(productId, quantity)
+  AddProduct(productIdentity, quantity = 1, data = {}) {
```

Comment:
@yasiloghmani one thing that might be a problem is if a customer is using a UUID as an SKU. In that case the request would fail as the code would send the SKU as an `id` 🤔 

We might have to have an optional parameter for `isSKU` 

`AddProduct(productIdentity, quantity = 1, data = {}, isSku = false)`

And if it is `true`, force `isProductId` to be false? (cc. @shaunmaharaj , @RyanKirby89 )

Reply:
I know @RyanKirby89 was fond of the config option too. If we use it, we don't need the regex IMO as we'll leave the capability to the user. Will need to update the docs snippet for this too.

I'm good either way but prefer the config option if it doesn't break compatibility.

Reply:
Please check https://github.com/moltin/js-sdk/pull/389 as well.

Reply:
@shaunmaharaj I am happy to stick with `isSku` if everyone else is good with it 👍

Reply:
I'm happy with isSku as well. @yasiloghmani can you please work with docs team to get the page updated to mention this config